### PR TITLE
Add basic conversion test.

### LINKED ISSUE #4: Create test scenario with Python/Pandas to verify conversion of Parquet to HDF5
Pandas would allow us to quickly create test cases, write them to Parquet, and then verify post conversion with `libsonata` that the content is identical.

### PR DESCRIPTION
**Pull request by matz-e**
Fixes #4.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,12 +27,23 @@ jobs:
...
+        cmake --install build
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+
+    - name: Small integration test
+      run: |
+        export PATH=$PATH:$PWD/install/bin
+        python -mpip install -r tests/test_requirements.txt
+        python tests/test_integration.py
```
Comment (1uc) [inline .github/workflows/test.yml:46]: It might pay off to immediately start using `pytest`.

File: tests/test_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+libsonata
+pandas
+pyarrow
```
Comment (1uc) [inline tests/test_requirements.txt:3]: is numpy missing?

File: tests/test_integration.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        pop = store.open_population(population_name)
+        assert len(pop) == len(df)
+
+        npt.assert_array_equal(
+            pop.get_attribute("my_attribute", pop.select_all()), df["my_attribute"]
+        )
+        npt.assert_array_equal(
+            pop.get_attribute("my_other_attribute", pop.select_all()),
+            df["my_other_attribute"],
+        )
```
Comment (1uc) [inline tests/test_integration.py:92]: The `*_node_id`, `edge_type_id` are not tested.

**Review by 1uc — DISMISSED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (matz-e) [reply .github/workflows/test.yml:46]: Well, it's already set up for that :)

File: tests/test_integration.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+
+def test_conversion():
+    with tempfile.TemporaryDirectory() as dirname:
+        tmpdir = Path(dirname)
+
+        parquet_name = tmpdir / "data.parquet"
+        parquet_name.mkdir(parents=True, exist_ok=True)
+        sonata_name = tmpdir / "data.h5"
+        population_name = "d-fault"
```
[the remark this answers is not in the document]
Comment (matz-e) [reply tests/test_integration.py:73]: Fair.

File: tests/test_requirements.txt (hunk 2 above)
Comment (matz-e) [reply tests/test_requirements.txt:3]: IIRC, both libsonata and pandas should require it.

File: tests/test_integration.py (hunk 3 above)
Comment (matz-e) [reply tests/test_integration.py:92]: I'll add tests for the `id` fields, but `edge_type_id` is not really exposed by `libsonata` IIRC

**Review by 1uc — APPROVED** [review]

**Pull request merged**